apps/io-wallet-user-func/src/infra/attestation-service/android/attestation.ts

```diff
@@ -46,7 +47,7 @@ export const verifyAttestation = async (params: VerifyAttestationParams) => {
     });
   }
 
-  validateIssuance(x509Chain, googlePublicKey);
+  validateIssuance(x509Chain, googlePublicKey, params.skipChainValidation);
```

Comment:
```suggestion
  validateIssuance(x509Chain, googlePublicKey, skipChainValidation);
```
with
`const { androidCrlUrl, googlePublicKey, httpRequestTimeout, x509Chain, skipChainValidation } = params;`

Reply:
Right, thanks you!